ipn/auditlog/auditlog_test.go

```diff
@@ -0,0 +1,484 @@
+// Copyright (c) Tailscale Inc & AUTHORS
+// SPDX-License-Identifier: BSD-3-Clause
+
+package auditlog
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"sync"
+	"testing"
+	"time"
+
+	"tailscale.com/ipn/store/mem"
+	"tailscale.com/tailcfg"
+	"tailscale.com/tstest"
+)
+
+func expectNoError(t *testing.T, err error) {
+	t.Helper()
+	if err != nil {
+		t.Fatalf("unexpected error: %v", err)
+	}
+}
+
+func expectError(t *testing.T, err error) {
+	t.Helper()
+	if err == nil {
+		t.Fatalf("expected error but got nil")
+	}
+}
```

Comment:
In the past when Go helper packages were proliferating in our tree, we decided in some eng meeting that https://github.com/go-quicktest/qt was our choice.  Maybe you could replace these two helpers with qt instead?